<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Основное окно</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#aaaaaa</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView_MainWindow">
        <property name="styleSheet">
         <string notr="true">background-color:#606060</string>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="11" column="0">
         <widget class="QPushButton" name="pushButton_Corel">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Corel/noCorel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит из таблицы
данные о аудиториях в
которых вместимость &gt; 25</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит из таблицы
только те группы если в
таблице группы есть одинаковая
специализация</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="Line" name="line_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="Line" name="line_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выбор таблицы</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит из таблицы фамилию,
имя, название группы и выводит
 только тех преподавателей которые
не привязаны к группе БСБО-03-19</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит номер, вид занятия,
максимальную вместимость аудитории</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="Line" name="line_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="pushButton_View">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_JoinCase">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Join+Case</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="Line" name="line_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит группу, кол-во студентов
специальность и преподавателя
к которому привязана группа</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QPushButton" name="pushButton_Any">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Any</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QPushButton" name="pushButton_Group">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Group/Having</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#606060; color:white; border-radius:2px</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_checkDB">
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Просмотр</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит id учителя
по введённому имени</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Выводит группу и количество студентов
по введёной специальности</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Prompt</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: 'Prompt', sans-serif;
text-align:center;
color:back;</string>
          </property>
          <property name="text">
           <string>Удалить из таблицы с расписанием строку</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_vector">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Специальность</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_id">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;</string>
          </property>
          <property name="placeholderText">
           <string>Id строки</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="Line" name="line_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButton_Vector">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Векторная функция</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="Line" name="line_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_Delete">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
color: #596a71;
font-family: 'Prompt', sans-serif;</string>
          </property>
          <property name="placeholderText">
           <string>Имя преподавателя</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="Line" name="line_7">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_Scalar">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
          </property>
          <property name="text">
           <string>Скаляр. функция</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_editDB">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;color:black;
font-family: 'Prompt', sans-serif;text-align:center;
font-weight:200</string>
        </property>
        <property name="text">
         <string>Окно для
редактирования</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
